server.go

```diff
@@ -126,6 +126,11 @@ func (s *Server) NewClientWithOptions(ctx context.Context, options client.Option
 	return client.NewClient(options)
 }
 
+// FrontendHostPort returns the host:port for this server.
+func (s *Server) FrontendHostPort() string {
+	return s.frontendHostPort
+}
+
```

Comment:
For your use case, would a method to return [`client.Options`](https://pkg.go.dev/go.temporal.io/sdk/client#Options) work as well?

Right now Temporalite really only needs a hostport to be set on the client, but in the future more client options may need to have modified default values and I would like to avoid confusion for end users about how to instantiate clients. I think this would make it possible to introspect things like hostport, but also decrease the chances that dependent code would break if/when Temporalite needs to further customize default client options.

Reply:
`client.Options` would work, though I might just be extracting the host:port out of it for some uses.